src/main/java/com/ncedu/cheetahtest/dao/testscenario/TestScenarioDaoImpl.java

```diff
@@ -0,0 +1,204 @@
+package com.ncedu.cheetahtest.dao.testscenario;
+
+import com.ncedu.cheetahtest.dao.genericdao.AbstractDaoImpl;
+import com.ncedu.cheetahtest.entity.testscenario.ItemDTO;
+import com.ncedu.cheetahtest.entity.testscenario.TestScenario;
+import com.ncedu.cheetahtest.exception.general.EntityNotFoundException;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.jdbc.core.JdbcTemplate;
+import org.springframework.stereotype.Repository;
+
+import java.util.List;
+
+import static com.ncedu.cheetahtest.dao.testscenario.TestScenarioConsts.*;
+
+@Repository
+public class TestScenarioDaoImpl extends AbstractDaoImpl<TestScenario> implements TestScenarioDao {
+
+    private static final String[] rows = {"id", "title", "description", "project_id", "status"};
+
+    @Autowired
+    public TestScenarioDaoImpl(JdbcTemplate jdbcTemplate) {
+        super(new TestScenarioMapper(), jdbcTemplate, rows, "test_scenario");
+    }
+
+    @Override
+    public TestScenario findById(int id) {
```

Comment:
You don't need to implement this method because it is already implemented in AbstractDaoImpl. You can call it using AbstractDao<TestScenario>.findById(int id).